packages/jsii/lib/compiler.ts

```diff
@@ -329,6 +374,16 @@ export class Compiler implements Emitter {
   }
 }
 
+/**
+ * A handle to a watch-mode operation.
+ */
+export interface Watch {
```

Comment:
Is this really needed?

Reply:
It's virtually impossible to unit test without that. Or there is no way to make the test finish.